deps/Makefile

```diff
@@ -272,6 +276,7 @@ include $(SRCDIR)/patchelf.mk
 include $(SRCDIR)/mbedtls.mk
 include $(SRCDIR)/libssh2.mk
 include $(SRCDIR)/libgit2.mk
+include $(SRCDIR)/curl.mk
```

Comment:
Good point. libgit2 and curl are in different build stages. Shouldn't that be good enough? I do think that libssh2 should probably be stage 1 (currently in 2), curl in stage 2 and libgit2 in stage 3.